requirements.txt

```diff
@@ -1,6 +1,9 @@
 Django==2.2
 django-authtools==1.7.0
 djangorestframework==3.12.2
-djangorestframework-simplejwt==4.6.0
+djangorestframework-simplejwt==4.4.0
```

Comment:
Let's test things with `djangorestframework-simplejwt==4.6.0`.
We should always try to use the newest versions.

Reply:
![swagger_in_use](https://user-images.githubusercontent.com/32267051/100496670-ff5fad00-317f-11eb-84d8-47d6e8f9faa4.png)